l10n_it_fatturapa/models/partner.py

```diff
@@ -71,25 +71,26 @@ def _check_ftpa_partner_data(self):
                         "As a natural person, partner %s "
                         "must have Name and Surname."
                     ) % partner.name)
-                if not partner.is_pa and (
-                    not partner.codice_destinatario or
-                    len(partner.codice_destinatario) != 7
-                ):
+                if not partner.is_pa and len(partner.codice_destinatario) != 7:
```

Comment:
Provato in runbot:

0. Crea nuovo partner
1. Metti nome e cognome
2. Spunta 'Subjected to Electronic Invoice'
3. Svuota 'Addressee code'
3. Salva:
```
Error while validating constraint

object of type 'bool' has no len()
```

Reply:
ok grazie, fixo! Mai fidarsi dei commit altrui :wink:

Reply:
Dovrei aver fixato.